### PR TITLE
Staging to PROD

### PR DESCRIPTION
**Pull request by prateek-giddh**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/invoice/generate/invoice.generate.component.scss [hunk 1]
```diff
@@ -181,16 +191,67 @@ button.btn.btn-filter {
...
+}
+
+.invoice-edit-wrapper {
+    height: 100vh;
+    overflow-y: auto;
+    .invoice-edit-header {
+        display: flex;
+        justify-content: space-between;
+        height: 54px;
+        background-color: #ffffff;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/invoice/generate/invoice.generate.component.scss:211]: use var color name

File: apps/web-giddh/src/app/invoice/generate/invoice.generate.component.scss [hunk 2]
```diff
@@ -181,16 +191,67 @@ button.btn.btn-filter {
...
+        border-top: 1px solid #c6c6c6;
+        border-bottom: 1px solid #c6c6c6;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/invoice/generate/invoice.generate.component.scss:213]: use var color name

File: apps/web-giddh/src/app/invoice/generate/invoice.generate.component.scss [hunk 3]
```diff
@@ -181,16 +191,67 @@ button.btn.btn-filter {
...
+            padding: 10px;
+            p {
+                font-size: 16px;
+                font-weight: bold;
+            }
+        }
+        span.close-modal {
+            height: 32px;
+            width: 32px;
+            background-color: #ddd;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/invoice/generate/invoice.generate.component.scss:225]: use var color name

File: apps/web-giddh/src/app/invoice/generate/invoice.generate.component.scss [hunk 4]
```diff
@@ -181,16 +191,67 @@ button.btn.btn-filter {
...
+    }
+}
+
+span.close-modal.cp.pull-right {
+    display: inline-grid;
+    justify-content: center;
+    align-items: center;
+    height: 32px;
+    width: 32px;
+    background-color: #ddd;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/invoice/generate/invoice.generate.component.scss:244]: use var color name

File: apps/web-giddh/src/app/invoice/generate/invoice.generate.component.scss (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/invoice/generate/invoice.generate.component.scss]: resolved

File: apps/web-giddh/src/app/invoice/generate/invoice.generate.component.scss (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/invoice/generate/invoice.generate.component.scss]: resolved

File: apps/web-giddh/src/app/invoice/generate/invoice.generate.component.scss (hunk 3 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/invoice/generate/invoice.generate.component.scss]: resolved

File: apps/web-giddh/src/app/invoice/generate/invoice.generate.component.scss (hunk 4 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/invoice/generate/invoice.generate.component.scss]: resolved

**Pull request merged**